### PR TITLE
Timofy

### PR DESCRIPTION
**Pull request by sergeyf**
Note: `tt verfiy` was run with `cuda: False` because s2-server2 is too old. It's still `False` in the config, but actually it should work fine with `cuda: True`

CC @kelseym-ai2

### CODE REVIEW

File: s2aff/timo/interface.py [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+    The fields below are examples only; please replace them with
+    appropriate fields for your model.
+
+    To learn more about declaring pydantic model fields, please see:
+    https://pydantic-docs.helpmanual.io/
+    """
+
+    raw_affiliation: str = Field(description="Raw affiliation string")
+    # field2: float = Field(description="Some other field of consequence")
```
Comment (cmwilhelm) [inline s2aff/timo/interface.py:33]: Can delete this line.

File: s2aff/timo/interface.py [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+            prediction_instance = Prediction(
+                ror_id=prediction["stage2_candidates"][0],
+                score=prediction["stage2_scores"][0],
+                main=prediction["main_from_ner"],
+                child=prediction["child_from_ner"],
+                address=prediction["address_from_ner"],
+            )
+        return prediction_instance
+
+    def predict_one(self, instance: Instance) -> Prediction:
```
Comment (cmwilhelm) [inline s2aff/timo/interface.py:121]: If your'e not calling through to `predict_one` from `predict_batch`, you can drop it altogether. The server code will always call `predict_batch`.

**Review by cmwilhelm — APPROVED** [review]
LGTM, 2 small comments

**Comment by cmwilhelm** [comment]
Ah @sergeyf . We chatted about this in slack, but I need to move a particular validation hook to `tt verify` that's only in `tt publish`. tldr: you need to add a `description=` metadata field to your `setup.py`.

File: s2aff/timo/interface.py (hunk 2 above)
Comment (sergeyf) [reply s2aff/timo/interface.py:121]: Maybe add this fact to the doc string for `predict_one`?

**Comment by sergeyf** [comment]
Thanks @cmwilhelm. I've addressed your comments.

**Review by comorado — APPROVED** [review]

**Comment by kelseym-ai2** [comment]
How was the gold test data set created? Like what was it sampled from?

**Comment by sergeyf** [comment]
@kelseym-ai2 the story of the gold data is sort of complicated and not well documented. In 2021, we did a first annotation for data I sampled from MAG in some <convoluted manner>. Then we decided we needed more data so I sampled more data from OpenAlex in <another convoluted manner>.

**Comment by sergeyf** [comment]
I have now published @cmwilhelm:

```
(s2aff_release) sergey@s2-server2.corp ~/S2AFF-release $ tt publish -c s2aff/timo/config.yaml
Publishing your package to pip.s2.allenai.org as s2aff==0.1...
Published successfully to: https://pip.s2.allenai.org/#/package/s2aff
Publishing your model config to TIMO Registry for s2aff==0.1...
```

**Pull request merged**